specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2019-04-01/containerregistry_build.json

```
@@ -0,0 +1,2455 @@
+          "description": "The name of the trigger.",
+          "type": "string"
+        }
+      }
+    },
+    "BaseImageTrigger": {
+      "description": "The trigger based on base image dependency.",
+      "required": [
+        "baseImageTriggerType",
+        "name"
+      ],
+      "type": "object",
+      "properties": {
+        "baseImageTriggerType": {
+          "description": "The type of the auto trigger for base image dependency updates.",
+          "enum": [
+            "All",
+            "Runtime"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "BaseImageTriggerType",
+            "modelAsString": true
+          }
+        },
+        "status": {
+          "description": "The current status of trigger.",
+          "enum": [
+            "Disabled",
+            "Enabled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "TriggerStatus",
+            "modelAsString": true
+          }
+        },
+        "name": {
+          "description": "The name of the trigger.",
+          "type": "string"
+        }
+      }
+    },
+    "SourceRegistryCredentials": {
+      "description": "Describes the credential parameters for accessing the source registry.",
+      "type": "object",
+      "properties": {
+        "loginMode": {
+          "description": "The authentication mode which determines the source registry login scope. The credentials for the source registry\r\nwill be generated using the given scope. These credentials will be used to login to\r\nthe source registry during the run.",
+          "enum": [
+            "None",
+            "Default"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "SourceRegistryLoginMode",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "CustomRegistryCredentials": {
+      "description": "Describes the credentials that will be used to access a custom registry during a run.",
+      "type": "object",
+      "properties": {
+        "userName": {
+          "$ref": "#/definitions/SecretObject",
+          "description": "The username for logging into the custom registry."
+        },
+        "password": {
+          "$ref": "#/definitions/SecretObject",
+          "description": "The password for logging into the custom registry. The password is a secret \r\nobject that allows multiple ways of providing the value for it."
+        },
+        "identity": {
+          "description": "Indicates the managed identity assigned to the custom credential. If a user-assigned identity\r\nthis value is the Client ID. If a system-assigned identity, the value will be `system`. In\r\nthe case of a system-assigned identity, the Client ID will be determined by the runner. This\r\nidentity may be used to authenticate to key vault to retreive credentials or it may be the only \r\nsource of authentication used for accessing the registry.",
```

retreive -> retrieve
```suggestion
          "description": "Indicates the managed identity assigned to the custom credential. If a user-assigned identity\r\nthis value is the Client ID. If a system-assigned identity, the value will be `system`. In\r\nthe case of a system-assigned identity, the Client ID will be determined by the runner. This\r\nidentity may be used to authenticate to key vault to retrieve credentials or it may be the only \r\nsource of authentication used for accessing the registry.",
```

---

@jaysterp could you commit the proposed changes?

---

Yes, see fix.